crossbar/adapter/rest/common.py

```diff
@@ -161,14 +168,16 @@ def _render_request(self, request):
             content_type_elements = []
 
         if self.decode_as_json:
-            # iff the client sent a content type, it MUST be one of _ALLOWED_CONTENT_TYPES
+            # if the client sent a content type, it MUST be one of _ALLOWED_CONTENT_TYPES
```

Comment:
ahh, I might replace it with the words then (since I don't come from a math background, so I thought it was a typo :) )